crates/bevy_color/src/lib.rs

```diff
@@ -4,6 +4,7 @@
     html_logo_url = "https://bevyengine.org/assets/icon.png",
     html_favicon_url = "https://bevyengine.org/assets/icon.png"
 )]
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
What does this line do?

Reply:
It removes the standard library if you _don't_ have the `std` feature enabled. The reason for the double negative is Rust features are supposed to be additive. Adding the `std` feature adds the standard library.

Reply:
Ah, so it resolves to #[no_std] when the std feature is not enabled.
I was confused what the attribute was relating to.